lib/Doctrine/ORM/EntityRepository.php

```diff
@@ -183,14 +181,12 @@ public function findAll()
     /**
      * Finds entities by a set of criteria.
      *
-     * @param array      $criteria
-     * @param array|null $orderBy
-     * @param int|null   $limit
-     * @param int|null   $offset
-     *
-     * @return array The objects.
+     * @param int|null $limit
+     * @param int|null $offset
      *
-     * @psalm-return list<T>
+     * @psalm-param array<string, mixed> $criteria
+     * @psalm-param list<string>|null $orderBy
```

Comment:
Not sure if I should create new issue for this, but after v2.8.3, Psalm throws a warning for this function in our codebase:

```
ERROR: InvalidArgument - src/SomeClass.php:25:13 - Argument 2 of Doctrine\ORM\EntityRepository::findBy expects list<string>\|null, array{id: "ASC"} provided (see https://psalm.dev/004)
12 | ['id' => 'ASC'],
```

I have a feeling this updated `@psalm-param` is not correct, because:
https://github.com/doctrine/orm/blob/2.8.x/lib/Doctrine/ORM/Persisters/Entity/BasicEntityPersister.php#L1145
```
@psalm-param array<string, string> $orderBy
```